fluidkeys/main.go

```diff
@@ -267,6 +270,63 @@ func keyCreate() exitCode {
 	return 0
 }
 
+func keyList() exitCode {
+	fluidkeysDirectory, err := getFluidkeysDirectory()
+	if err != nil {
+		fmt.Printf("Failed to get fluidkeys directory")
+	}
+	db := database.New(fluidkeysDirectory)
+
+	keys, err := loadPgpKeys(db)
+	if err != nil {
+		panic(err)
+	}
+
+	emailAddressColumnWidth := len("Email address")
```

Comment:
So what happens if I have an email address longer than the string `Email address` ? We talked about searching for the longest email address on the key, did that happen?